repo/packages/A/artifactory/2/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "packagingVersion": "3.0",
+  "name": "artifactory",
+  "version": "5.1.0",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/JfrogDev/artifactory-dcos.git",
+  "maintainer": "soleng-team@jfrog.com",
+  "website": "https://jfrog.com",
+  "framework": false,
+  "description": "As the first, and only, universal Artifact Repository Manager on the market, JFrog Artifactory fully supports software packages created by any language or technology.",
+  "tags": [
+    "mesosphere",
+    "framework",
+    "artifacts",
+    "artifactory"
+  ],
+  "preInstallNotes": "If you wish to use Artifactory High Availability, You will need atlead 1 public slave in your cluster",
```

Comment:
`atlead` -> `at least`
`slave` -> `agent`

Don't you currently need a public agent also for Artifactory Pro?